packages/web-app-files/src/store/actions.ts

```diff
@@ -390,7 +390,7 @@ export default {
     return Promise.allSettled(shareQueriesPromises).then(() => {
       context.commit(
         'OUTGOING_SHARES_SET',
-        shares.filter((s) => s.outgoing)
+        shares.filter((s) => s.outgoing && s.collaborator.name !== undefined)
```

Comment:
This will break link shares because they don't have a `collaborator` prop. Their name sit's in `s.name`.